docs/platforms/android/releases/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Release Channels
+description: "Learn about the Android SDK's release channels and choose the right one."
+sidebar_order: 7000
+---
+
+The Android SDK uses the following channels to release SDK updates:
+
+## `Pre-release`
+
+Pre-releases (alpha, beta, RC,…) are for larger and potentially more impactful changes, such as new features or major versions.
+
+## `Latest`
+
+We continuously release major/minor/hotfix versions from the `main` branch. These releases go through all our internal quality gates and are very safe to use. They're intended to be the default for most teams.
+
+## `Stable`
+
+Releases are promoted from `Latest` to `Stable` when they've been used in the field for some time and at some scale. The length of time a release has been out, the level of adoption, and other quality and stability metrics are considered. These releases will be indicated on the releases page (https://github.com/getsentry/sentry-cocoa/releases/) with the `Stable` label.
+
+## FAQ
+
+### Where can I find the latest stable release for any given SDK?
+
+On the releases pages of the SDK's GitHub repo. Stable releases will be indicated with the label `(Stable)` in the name of the release. You can also use the search box in the top right to search for the newest releases.
+
+### Does this mean the release channels are not stable?
+
+No. Stability and performance are the top priorities for every release. The `Stable` release status means that the release has been tested and proven to work in the wild and at scale.
+
+### How often do we mark SDKs as stable?
+
+The target cadence is to provide stable releases every month.
```

Comment:
when every month?

Reply:
no fixed date. we "promote" existing releases that have been out for a while when it makes sense, and we plan to do this at least once a month, but it can depend on release schedule (e.g. prior to a new major there might be fewer stables for the current major)